### PR TITLE
Firefox 45 compat

### PR DESCRIPTION
**Pull request by EnTeQuAk**

### CODE REVIEW

File: tests/compat/test_gecko45.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
+from helper import CompatTestCase
+from validator.compat import FX45_DEFINITION
+
+
+class TestFX44Compat(CompatTestCase):
+    """Test that compatibility tests for Gecko 44 are properly executed."""
```
Comment (diox) [inline tests/compat/test_gecko45.py]: Copy/pasting on the name/docstring : that "44" needs to be "45".

File: validator/app_versions.json [hunk 2]
```diff
@@ -63,7 +63,10 @@
          "41.0a1", "41.0a2", "41.0", "41.*",
          "42.0a1", "42.0a2", "42.0", "42.*",
          "43.0a1", "43.0a2", "43.0", "43.*",
-         "44.0a1", "44.0a2", "44.0", "44.*"]
+         "44.0a1", "44.0a2", "44.0", "44.*",
+         "45.0a1", "45.0a2", "45.0", "45.*",
+         "46.0a1", "46.0a2", "46.0", "46.*"]
```
Comment (diox) [inline validator/app_versions.json]: The "46" part should probably be left for #399

Comment (EnTeQuAk) [inline validator/app_versions.json]: Ah yeah, I wanted to do them into one PR first so I put this part here and have the same change in both PRs.

**Comment by diox** [comment]
r+

**Pull request merged**